addons/mail_bot/views/discuss.xml

```diff
@@ -10,8 +10,15 @@
         @param {mail.NotificationAlert} widget
     -->
     <t t-name="mail.NotificationAlert">
-       <center t-if="widget.isNotificationBlocked" class="o_notification_alert alert alert-primary">
-           Odoo Push notifications have been blocked. Go to your browser settings to allow them.
-       </center>
+        <div class="o_NotificationAlert alert alert-primary" t-if="widget.isNotificationBlocked">
+            Odoo Push notifications have been blocked. Go to your browser settings.
+            <span t-if="widget.settingUrl" class="border border-primary rounded o_NotificationAlert_url">
+                <t t-esc="widget.settingUrl"/>
+                <button class="btn btn-sm btn-primary rounded-0 o_NotificationAlert_urlClipboardButton">
+                    <span class="fa fa-clipboard"></span><span> Copy Link</span>
+                </button>
```

Comment:
I think text node as sibling to a HTML node may make things harder for proper style, which is certainly the reason that you used absolute positioning. I would wrap text node in its own HTML node (e.g. `<span>`) to prevent such issues.